test/image/mocks/gl3d_isosurface_out_of_iso_range_case.json

```diff
@@ -0,0 +1,88 @@
+{
+    "data": [{
+        "type": "isosurface",
+        "colorscale": "Portland",
+        "reversescale": false,
+        "surface": { "show": true, "fill": 0.8 },
+        "spaceframe": { "show": false },
+        "slices": {
+            "x": { "show": true, "fill": 1, "locations": [-10] },
+            "y": { "show": true, "fill": 1, "locations": [-10] },
+            "z": { "show": true, "fill": 1, "locations": [-10] }
+        },
+        "caps": {
+            "x": { "show": false },
+            "y": { "show": false },
+            "z": { "show": false }
+        },
+        "contour": {
+            "show": true,
+            "width": 4
+        },
+        "x": [ -10.000, -7.143, -4.286, -1.429, 1.429, 4.286, 7.143, 10.000 ],
+        "y": [ -10.000, -6.000, -2.000, 2.000, 6.000, 10.000 ],
+        "z": [ -10.000, -3.333, 3.333, 10.000 ],
+        "isomin": -1010,
+        "isomax": 1010,
+        "value": [
```

Comment:
Oh. Sorry I didn't catch this sooner. `value` should have the same length as `x`, `y`, `z` so that the `isosurface` api matches the one for `cone` and `streamtubes`.

In order words, `x`, `y` and `z` must be set with repeated coordinates. See for example [`gl3d_streamtube-simple.json`](https://github.com/plotly/plotly.js/blob/master/test/image/mocks/gl3d_streamtube-simple.json).

We could also support "non-repeated" x/y/z signature, but `value` in this case should be 3D (i.e. triply-nested) similar to how `heatmap` and `contour` support 1D and 2D `z` arrays.

Reply:
OK. Am fixing that...